pf/tfbridge/provider_check.go

```diff
@@ -46,6 +46,15 @@ func (p *provider) CheckWithContext(
 		return checkedInputs, []plugin.CheckFailure{}, err
 	}
 
+	if info := rh.pulumiResourceInfo; info != nil {
```

Comment:
This bypasses defaults.ApplyDefaultInfoValues and p.validateResourceConfig and tfbridge.MarkSchemaSecrets potentially dropping the features implemented in this code. Would it be worthwhile to figure out how to express the transformation in a way that still calls into that code and keeps the features? Perhaps something along the lines of PreConfigureCallback like PreCheckCallback?

Reply:
Ah never mind. It keeps those in place.